### PR TITLE
Update opcodes for 6.31 hotfix

### PR DESCRIPTION
**Pull request by Aireil**
Now with 0x added, big hype!!

### CODE REVIEW

File: UIRes/serveropcode.json [hunk 1]
```diff
@@ -1,14 +1,14 @@
 {
-  "ActorControlSelf": 615,
-  "HousingWardInfo": 221,
```
Comment (reiichi001) [inline UIRes/serveropcode.json:3]: Has HousingWardInfo been confirmed? According to <https://github.com/karashiiro/FFXIVOpcodes/>, it's `685` in dec, which is `0x2AD` in hex. That one doesn't get listed on the Ipcs.cs though, online in the decimal formatted json.

**Review by reiichi001 — COMMENTED** [review]
Why hex, when we've historically done decimal? If we're going to, I'd recommend upper case letters for readability.

File: UIRes/serveropcode.json (hunk 1 above)
Comment (Aireil) [reply UIRes/serveropcode.json:3]: Not sure, going to check, I was just looking in the .cs and didn't see it, didn't know it was in the json.

Comment (Aireil) [reply UIRes/serveropcode.json:3]: Hmm, this looks fine? Not sure how else to test it though.
image

**Pull request merged**